spec/ttt/player_spec.clj

```diff
@@ -1,18 +1,19 @@
 (ns ttt.player-spec
-  (:require [speclj.core :refer :all]
-            [ttt.player :refer :all]))
+  (:require [speclj.core :refer [describe should= context it]]
+            [ttt.player :as player]))
 
 (describe "Player"
   (context "#players"
     (it "creates a player with marker X"
-      (should= "X" (:marker (first players))))
+      (should= "X" (:marker (first player/players))))
     (it "creates a player with marker O"
-      (should= "O" (:marker (second players)))))
+      (should= "O" (:marker (second player/players)))))
```

Comment:
I think instead of creating two tests for this, we can combine them into one and assert on whatever `player/players` returns.

Reply:
This is a good example of "one _logical_ assertion, per test".  You could think of saying "it creates enough players to play the game" as the behavior that's meaningful to humans and the two `should=` checks inside it.  The two assertions would still be written–and the production code driven–by separate red-green-refactor cycles.